cli/sdl-brew.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 # Set up environment for building with SDL found in homebrew.
```

Comment:
The 'brew' in this file is referring to the Mac 'homebrew' system, so it doesn't make sense to have it support Linux too.  I think you could make a separate Linux script if we need it.

...but also, I would assume LIBRARY_PATH already includes /usr/lib on Linux, shouldn't it?

Reply:
I was doing this from WSL, I didn't check to see what it was on my actual linux box. But I didn't see any default for `LD_LIBRARY_PATH` or `LIBRARY_PATH` I did see that there is a homebrew for linux, but didn't want to install that  myself when the package itself could easily be installed with `apt` (or another package manager of choice)